### PR TITLE
rui-horizontal-nav

### LINKED ISSUE #54: Horizontal Tab Component

### PR DESCRIPTION
**Pull request by zulaica**
Fixes #54

### CODE REVIEW

File: addon/components/rui-nav-horizontal-link/component.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  classNames: ['nav-link'],
+  positionalParams: 'params',
+
+  init: function() {
+    // this._super.apply(this, arguments);
+    this._super(...arguments);
+    Ember.Logger.log('Attrs: ' + this.get('attrs'));
+    Ember.Logger.log('Params: ' + this.get('params'));
+    Ember.Logger.log('Classes are ' + this.get('classNames'));
+  },
```
Comment (kscherling) [inline addon/components/rui-nav-horizontal-link/component.js]: What's all this for?  
Maybe a better way is just to put a simple link-to helper in the *-item component and yeild into it passing the route in.  much simpler plus 3 components for a simple tab seems a bit overkill.

Comment (zulaica) [inline addon/components/rui-nav-horizontal-link/component.js]: Good call. I'll get that sorted today.

**Comment by zulaica** [comment]
@kscherling: This should be much better now.

File: app/components/rui-nav-horizontal-link/component.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default } from 'ember-cli-revelation-ui/components/rui-nav-horizontal-link/component';
```
Comment (kscherling) [inline app/components/rui-nav-horizontal-link/component.js]: Probably don't need this anymore...

**Pull request merged**